rusk-wallet/src/bin/interactive/command_menu.rs

```diff
@@ -185,7 +192,7 @@ pub(crate) async fn online(
             ProfileOp::Run(Box::new(Command::Stake {
                 address: Some(addr),
                 owner: Some(owner),
-                amt: prompt::request_stake_token_amt(balance)?,
+                amt: prompt::request_stake_token_amt(balance, has_stake)?,
```

Comment:
Why don't you choose the prompt according to the fetch stake?
If some, you can ask for normal amount, otherwise you can request the stake amount (with 1000minimi)

This way you:
- can remove the need for the "has_stake" variable
- avoid to change existing method

Reply:
I'm afraid that can be more convoluted, we want to call the prompt function in sequence so the code will become messy if I take that route I will document the has_stake